### ISSUE TITLE
check_mailq doesn't recognize -W and -C arguments.

### ISSUE THREAD

**Issue by CeeTeih0** [monitoring-plugins/monitoring-plugins#1627]
Hi there! Either the documentation [1] is outdated or check_mailq is missing optional arguments.
```
./check_mailq -w 100 -c 200 -W 10 -C 20
Unknown option: W
Unknown option: C
```
I tested the version from the master branch.

[1]  https://www.monitoring-plugins.org/doc/man/check_mailq.html

**Comment by RincewindsHat** [comment]
Can confirm for Debian stable

**Comment by RincewindsHat** [comment]
This seems to be an artifact from https://github.com/monitoring-plugins/monitoring-plugins/issues/381 . This was fixed in aa79ab278ffc9bf3d41207d97bdfb04be35b5c74 .
This issue can be closed I guess.
